### PR TITLE
Fix predict covariance

### LINKED ISSUE #972: EKF SLAM predict
https://github.com/AtsushiSakai/PythonRobotics/blob/73d1189e824c38af70f724d3ba5b0322e55f142d/SLAM/EKFSLAM/ekf_slam.py#L33

It seems to me that the input for `jacob_motion` should be `xEst` instead of `xEst[0:S]`, and the dimension of `G` should be len(xEst) by len(xEst).

https://github.com/AtsushiSakai/PythonRobotics/blob/73d1189e824c38af70f724d3ba5b0322e55f142d/SLAM/EKFSLAM/ekf_slam.py#L35

This line updates only the covariance of the pose; it does not update the covariance between the pose and the map, i.e. $\Sigma_{xm}$.

### PR DESCRIPTION
**Pull request by Chris7462**
#### Reference issue
Fix #972 

#### CheckList
- [ ] Did you add an unittest for your new example or defect fix?
- [ ] Did you add documents for your new example?
- [ ] All CIs are green? (You can check it after submitting)

### CODE REVIEW

File: SLAM/EKFSLAM/ekf_slam.py [hunk 1]
```diff
@@ -30,9 +30,9 @@
 def ekf_slam(xEst, PEst, u, z):
     # Predict
     S = STATE_SIZE
```
Comment (AtsushiSakai) [inline SLAM/EKFSLAM/ekf_slam.py:32]: Can we remove S?

**Comment by AtsushiSakai** [comment]
Als, is it possible the EKF SLAM doc too? https://github.com/AtsushiSakai/PythonRobotics/blob/master/docs/modules/slam/ekf_slam/ekf_slam_main.rst

File: SLAM/EKFSLAM/ekf_slam.py (hunk 1 above)
Comment (Chris7462) [reply SLAM/EKFSLAM/ekf_slam.py]: Certainly, `S` is used to denote the innovation covariance in the update step, so using `STATE_SIZE` in the predict step enhances code clarity.

**Comment by Chris7462** [comment]
> Als, is it possible the EKF SLAM doc too? https://github.com/AtsushiSakai/PythonRobotics/blob/master/docs/modules/slam/ekf_slam/ekf_slam_main.rst

Done. I have also removed all the trailing white space.

File: SLAM/EKFSLAM/ekf_slam.py [hunk 2]
```diff
@@ -197,7 +196,7 @@ def pi_2_pi(angle):
 
 
 def main():
-    print(__file__ + " start!!")
```
Comment (AtsushiSakai) [inline SLAM/EKFSLAM/ekf_slam.py:200]: Why did you remove the file name?

**Review by AtsushiSakai — APPROVED** [review]
LGTM
